### PR TITLE
Pl 144 toolbar design changes

### PR DESCRIPTION
**Pull request by dariomesic**
Done:

- design for navbar
- "title" of pdf
- removed toolbar UI component
- text on welcome page is RETURNED and FIXED so it MATCHES

TODO( in some other PR i guess ):
- zoom level of the page so it can fit in input field

### CODE REVIEW

**Comment by gabrielekarra** [comment]
title is not centered with the button to open the preview of others pdfs and i think "onew new pdf" has to be a button, maybe i will put it on the left before every button. zoom % doesn't change with button

Design is amazing now, good job!

File: cypress/e2e/pdfReader.cy.js [hunk 1]
```diff
@@ -10,6 +10,5 @@ describe("PDF Legacy pdf reader basic tests", () => {
   it("Pdf reader UI", () => {
     cy.title().should("eq", "PDFL - PDF Legacy");
 
-    cy.get("#sidenav").children().should("have.length", 5);
```
Comment (tonesija) [inline cypress/e2e/pdfReader.cy.js:13]: Add something similar to this line so it passes, Not just delete it.

**Review by tonesija — COMMENTED** [review]
There is a problem when opening a pdf, for 0.5 seconds the css does not load somehow. I don't know why is that.
 - this will probably be fixed in production, because css config is different there.

Tooltips go off screen for fullscreen and help buttons.

**Comment by tonesija** [comment]
+1 from me, the tooltips if you can do them great, if not, its okay

**Comment by dariomesic** [comment]
> +1 from me, the tooltips if you can do them great, if not, its okay

i ll do them dw

**Comment by paolocorsa** [comment]
Good job Dario it is amazing.
For me is +1.

**Comment by gabrielekarra** [comment]
this can be merged

**Pull request merged**